### PR TITLE
added the two projects newly and updated the readme

### PR DESCRIPTION
**Pull request by seabeePraveen**
After considering the checks so i have removed the secret key values form the project and made this new PR

### CODE REVIEW

File: Projects/Django-Projects/EduFlow/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+* Teacher information management: Manage teacher profiles, including personal information, teaching experience, and qualifications.
+* Course management: Create and manage courses, assign teachers and students to them.
+* Gradebook management: Manage and record student grades, create gradebooks and input grades electronically.
+* Fee management: Manage student fees, generate invoices, accept online payments, and track payment history.
+* Library management: Manage the school library, check out and return books, track inventory, and place holds.
+
+## ER Diagram
+<img src="./ER_Map.png" />
+
+## Getting Started
```
Comment (sree-hari-s) [inline Projects/Django-Projects/EduFlow/README.md:15]: modify getting started for better understanding

**Comment by sree-hari-s** [comment]
create a file `.env_sample` and create a sample of the original .env file.

and modify the instructions in the `Readme.md` for Eduflow

**Comment by sree-hari-s** [comment]
In todolist `requirements.txt` `Readme.md` file missing

**Comment by seabeePraveen** [comment]
yeah okay for the sample did u mean
.env_sample
SECRET_KEY = <your_key>

like this??

**Review by sree-hari-s — CHANGES_REQUESTED** [review]
Please make the modifications.

**Comment by sree-hari-s** [comment]
> yeah okay for the sample did u mean .env_sample SECRET_KEY = <your_key>
> 
> like this??

Yeah just like that also add that in an instruction as it was not specified there.

**Comment by seabeePraveen** [comment]
hello @sree-hari-s please once review this and let me know
Thank you!

**Comment by sree-hari-s** [comment]
> hello @sree-hari-s please once review this and let me know Thank you!

Eduflow is working fine 

image

it would be better if you could resolve blank tasks and also edit tasks would be a nice feature.

**Comment by seabeePraveen** [comment]
yeah okay u just want me to undo the those empty tasks right? like it shouldn't take the input task as input and add into database right?

**Comment by sree-hari-s** [comment]
> yeah okay u just want me to undo the those empty tasks right? like it shouldn't take the input task as input and add into database right?

empty input should not be accepted that's all.

**Comment by seabeePraveen** [comment]
okay

**Pull request merged**